### PR TITLE
Outline button red text

### PR DESCRIPTION
**Pull request by jenigooley**
**Added** outline red button styles 
Juice docs:
screen shot 2017-12-23 at 7 55 21 pm
screen shot 2017-12-23 at 7 55 14 pm
screen shot 2017-12-23 at 7 55 06 pm

### CODE REVIEW

File: docs/partials/buttons.html [hunk 1]
```diff
@@ -94,6 +101,10 @@ <h3>With icons</h3>
     <div class="button-wrapper">
       <button class="button-medium-outline-with-icon">Mark as done</button>
     </div>
+    <div class="button-wrapper">
+      <button class="button-medium-outline-red-with-icon">Mark as done</button>
+    </div>
+     <div c
```
Comment (sashako) [inline docs/partials/buttons.html]: what's going on with that tag? rm?

File: docs/styles/_buttons-section.scss [hunk 2]
```diff
@@ -36,7 +36,7 @@
     }
   }
 
-  $buttonTypes: ('blue', 'green', 'red', 'outline', 'link', 'link-gray');
+  $buttonTypes: ('blue', 'green', 'red', 'outline', 'outline-red', 'link');
```
Comment (sashako) [inline docs/styles/_buttons-section.scss]: why did you remove link-gray?

File: docs/partials/buttons.html [hunk 3]
```diff
@@ -168,7 +194,13 @@ <h4>loading</h4>
       <br />
       <button disabled class="button-medium-outline loading">Deny Request</button>
       <br />
-      <button  disabled class="button-small-outline-with-icon loading">Mark as done</button>
```
Comment (sashako) [inline docs/partials/buttons.html]: is that an intentional remove?

File: docs/partials/buttons.html [hunk 4]
```diff
@@ -169,7 +168,7 @@ <h4>disabled</h4>
       <button disabled class="button-medium-outline disabled">Deny Request</button>
       <br />
       <button  disabled class="button-small-outline-with-icon disabled">Mark as done</button>
-      <br />
+      <br />
```
Comment (finalprototype) [inline docs/partials/buttons.html]: nit: random trailing space
